wiki/models/urlpath.py

```diff
@@ -110,7 +112,7 @@ def delete_subtree(self):
     
     @classmethod
     def root(cls):
-        site = Site.objects.get_current()
+        site = get_current_site(get_current_request())
```

Comment:
yes, the implementation logic is
1. first look at the `SITE_ID` if present, return `Site` for that
2 . if `SITE_ID` is not defined then pick Site based on the request
3. if there is no `Site` for the current request, return `RequestSite` object for the request.